x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.ts

```diff
@@ -106,6 +106,25 @@ export const updatePackagePolicy = (
   return packagePolicyService.update(soClient, esClient, packagePolicy.id, updatedPackagePolicy);
 };
 
+export const setRulesOnPackage = async (
+  packagePolicyService: PackagePolicyServiceInterface,
+  packagePolicy: PackagePolicy,
+  esClient: ElasticsearchClient,
+  soClient: SavedObjectsClientContract
+) => {
+  const cspRules = await getCspRules(soClient, packagePolicy);
+  const rulesConfig = createRulesConfig(cspRules);
+  const dataYaml = convertRulesConfigToYaml(rulesConfig);
+
+  return await updatePackagePolicy(
```

Comment:
nit: I find it hard to understand the common sense in how code is being divided into different methods
There are simple principles we can follow to align in the way we should do it. 
One of them is [Single Responsibility Principle](https://stackify.com/solid-design-principles/)

in the example here, `updatePackagePolicy` is also `setVarToPackagePolicy` and saving the package into a persistent storage. which contradicts the single responsibility principle. 
IMO it would be better to call `setVarToPackagePolicy` before calling `updatePackagePolicy`

Going that direction, I don't see the reason to wrap `packagePolicyService.update` with another method.